### PR TITLE
docs(plugin-ideal-image): mention plugin changes type of imported images

### PR DESCRIPTION
**Pull request by samatt14**
Included information about changes to the types of imported support images

## Pre-flight checklist

- [ ] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/docusaurus-plugin-ideal-image/README.md [hunk 1]
```diff
@@ -5,3 +5,14 @@ Docusaurus Plugin to generate an almost ideal image (responsive, lazy-loading, a
...
+
+### Important 
+
+This plugin changes the type of imported supported images.
+
+e.g.
+
+Before: string
+
+After: {preSrc: string, src: import("@theme/IdealImage").SrcImage}
```
Comment (slorber) [inline packages/docusaurus-plugin-ideal-image/README.md:17]:
```suggestion
### Important 

This plugin registers a [Webpack loader](https://webpack.js.org/loaders/) that changes the type of imported/require images:

- Before: `string`
- After: `{preSrc: string, src: import("@theme/IdealImage").SrcImage}`
```

**Review by slorber — APPROVED** [review]
Thanks, some users will find it useful

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Please update the document in the website as well, because this one is not particularly discoverable.

**Comment by slorber** [comment]
> Thanks, some users will find it useful

will do

**Review by Josh-Cena — APPROVED** [review]
👍

We can even remove the README change if you agree, because it's a single link to the website anyway.

**Pull request merged**